src/main/java/uk/gov/beis/els/util/FontUtils.java

```diff
@@ -0,0 +1,61 @@
+    );
+
+
+    // Character widths are relative to font size in pixels. For example, if a character is 10px wide when the font
+    // size is 10px, the character width is 1. If it's 7px wide when the font size is 10px, the character width is 0.7
+    private final Map charWidths;
+
+    FontProperties(Map charWidths) {
+      this.charWidths = charWidths;
+    }
+
+
+    public Map getCharWidths() {
+      return this.charWidths;
+    }
+
+    public static FontProperties fromFontFaceName(String fontFaceName) {
+      switch(fontFaceName){
+        case "Verdana Regular":
+          return FontProperties.VERDANA_REGULAR;
+        case "Verdana Bold":
+          return FontProperties.VERDANA_BOLD;
+        case "Calibri Regular":
+          return FontProperties.CALIBRI_REGULAR;
+        default:
+          throw new RuntimeException(String.format("Unknown font face name '%s' while looking up FontProperties", fontFaceName));
+      }
+    }
+  }
+
+  public static Float estimatedTextWidth(String text, Float fontSize, FontProperties font) {

```

Comment:
Nit: can use the primitive `float` as the return type and font size arg, since we know they are non-null